src/afl-common.c

```diff
@@ -518,6 +518,135 @@ int parse_afl_kill_signal_env(u8 *afl_kill_signal_env, int default_signal) {
 
 }
 
+#define HELPER_MIN3(a, b, c) \
+  ((a) < (b) ? ((a) < (c) ? (a) : (c)) : ((b) < (c) ? (b) : (c)))
```

Comment:
Technically an incorrect/dangerous macro (`MIN3(1, 2, i++)`), could be rewritten als block in Gnu C, but likely fine if we just don't abuse it.

Reply:
done inline